content/geoip/updating-databases.md

```diff
@@ -88,7 +88,15 @@ EditionIDs YOUR_EDITION_IDS_HERE
 Run `geoipupdate`. To fully automate this process on Linux or Unix, use a
 crontab file like:
 
-<GeoIPCrontab />
```

Comment:
This component/shortcode is used because I needed some custom HTML.  I was thinking partials would work, but afaict they can only be used in templates, not content files.

Reply:
Sorry I mean like the actual Gatsby code. I don't see the `<GeoIPCrobtab />` / `src/components/GeoIPCrontab` used anywhere except the updating geoip databases markdown file.

Reply:
Ah, I planned on deleting the whole `src/components` folder once we were close to release.  But I can delete this specific component now as it really doesn't harm anything; in fact, everything under `src/components` is technically unused now that we're using Hugo, but keeping it around for now is useful as we migrate components to shortcodes.

Reply:
I agree it would be good to remove components as we migrate them. We could probably also delete where we import `<GeoIPCrontab />` in  https://github.com/maxmind/dev-site/blob/3fa29a3d3fa30f6fca1444953ddbfc0ef887e1bb/src/components/Mdx/index.ts#L9